ansible/roles/products/common/scene-files/defaults/main.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+---
+scene_file: ""
+scene_file_left: ""
+scene_file_right: ""
+scene_file_bi: ""
```

Comment:
How much effort would that bi to turn `bi` to `bimanual`? Would be much more maintainable and understandable

Reply:
done